src/components/ReactGoogleMaps/ReactGoogleMaps.js

```diff
@@ -181,39 +181,19 @@ export const ReactGoogleMaps = ({ google }) => {
   }, [allResources.length, dispatch]);
 
   useEffect(() => {
-    const setDefaultLocation = () => {
-      setCurrentLat(parseFloat(CITY_HALL_COORDINATES.latitude));
-      setCurrentLon(parseFloat(CITY_HALL_COORDINATES.longitude));
+    const fetchCoordinates = async () => {
+      try {
+        const position = await getCoordinates();
+        setCurrentLat(position.coords.latitude);
+        setCurrentLon(position.coords.longitude);
+      } catch (error) {
+        console.error("Error obtaining geolocation. Default coordinates will be used.");
```

Comment:
Can we noop this catch statement? We don't get much value from these console statements

Reply:
If we yeet the catch, we'll get an unhandled GeolocationPositionError from `getCoordinates`. I removed the console statement in the latest commit though
<img width="320" alt="Screen Shot 2024-08-11 at 7 59 16 PM" src="https://github.com/user-attachments/assets/10da4357-52c7-41ee-a621-8372fe1e9860">